code/pizzabot.js

```diff
@@ -1,7 +1,98 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+// 1. Definition of variables 
 
+const vegetarian = "Vegetarian"
+const hawaiian = "Hawaiian"
+const pepperoni = "Pepperoni"
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+
+// 2. Functions
+
+// Function for validating ordername
+const validateOrderName = (orderName) => {
+    if (orderName.toLowerCase() === vegetarian.toLowerCase()) {
+        return true;
+    } else if (orderName.toLowerCase() === hawaiian.toLowerCase()) {
+        return true;
+    } else if (orderName.toLowerCase() === pepperoni.toLowerCase()) {
+        return true;
+    } else {
+        alert('Select a pizza from the menu!');
+        return false;
+    }
+};
+
+// Function for calculating total cost
+const calculateTotalCost = (orderQuantity, pizzaPrice) => {
+    let orderTotal = orderQuantity * pizzaPrice;
+    return orderTotal;
+};
+
+// Function for calculating total cooking time
+const calculateCookingTime = (orderQuantity) => {
+    let orderTime = 0; 
+    if (orderQuantity <= 2) {
+        orderTime = 10;
+        return orderTime;
+
+    } else if (orderQuantity < 6) {
+        orderTime = 15;
+        return orderTime;
+
+    } else {
+        orderTime = 20;
+        return orderTime;
```

Comment:
Great job :)
Regarding clean code, there is also another convention, let me introduce you to it.
Apart from returning given thing from function, `return` keyword does one more thing - it automatically finishes function. That's why we mustn't write anything below return, because that code will never be executed (of course in your case we have code below `return true`, but it's inside if statement, so it wouldn't enter second statement in the first place).
Thanks to that feature of `return` keyword, we can write our function in slightly different, far more clean fashion :
```
const calculateCookingTime = (orderQuantity) => {
    let orderTime = 0; 
    if (orderQuantity <= 2) {
        orderTime = 10;
        return orderTime;

    } else if (orderQuantity < 6) {
        orderTime = 15;
        return orderTime;
    }
    orderTime = 20;
    return orderTime;
}
```
As you can see, we are not writing `else` statement anymore. We can do that thanks to behaviour mentioned above; if result of if statement is lower than 6, we will execute code from one of two first conditionals and finish function. If, on the other hand, result of it is equal or higher than 6, we will never enter any of the `if` conditionals and execute only two last lines of this functon (so, `orderTime = 20` and `return orderTime`). After executing them, we will finish function same way :)